runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationSections.module.scss

```diff
@@ -0,0 +1,160 @@
+@import "#src-landing/scss/main.scss";
+
+$ARROW_ICON_WIDTH: 300px;
+
+.root {
+  background-color: $palette-accent-primary;
+  position: relative;
+}
+
+.arrowIconWrapper {
+  width: $ARROW_ICON_WIDTH;
+  position: absolute;
+}
+
+.arrowIcon {
+  width: 100%;
+  height: 100%;
+  filter: brightness(0) invert(1);
+}
+
+.centeredWrapper {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-top: $padding-10;
+  padding-bottom: $padding-10;
+  padding-left: $padding-3;
+  padding-right: $padding-3;
+}
+
+.title {
+  @include font-face-base;
+  color: $text-color-light;
+  font-weight: $font-weight-bold;
+  font-size: $font-size-h2;
+}
+
+.fontDark {
+  color: $text-color-dark;
+}
+
+.logosGrid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: $padding-4;
+  padding-top: $padding-3;
+  padding-bottom: $padding-3;
+}
+
+.logosGrid div {
+  box-sizing: border-box;
```

Comment:
isn't `border-box` in global styles for every tag?

Reply:
it does. removed. done